projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -0,0 +1,262 @@
+                </div>
+                <div *ngFor="let cart of quotes?.carts" class="col-4">
+                  {{
+                    ((cart?.deliveryOrderGroups)[0]?.entries)[0]?.product
+                      ?.cartDisplayName
+                  }}
+                </div>
+              </li>
+              <ng-container
+                *ngTemplateOutlet="
+                  generalInformationQuoteComparison;
+                  context: {
+                    labelList: ((quotes?.carts)[0]?.entries)[0]?.product
+                      ?.configurable
+                      ? (((quotes?.carts)[0]?.entries)[0]
+                          ?.configurationInfos)[0]?.configurationValues?.entry
+                      : ((quotes?.carts)[0].insuranceQuote?.insuredObjectList
+                          ?.insuredObjects)[0]?.insuredObjectItems,
+                    configurable:
+                      ((quotes?.carts)[0]?.entries)[0]?.product?.configurable
+                  }
+                "
+              >
+              </ng-container>
+              <li class="accordion-list-item row sub-title">
+                <div class="col-4">
+                  {{ 'quote.selectQuoteNo' | cxTranslate }}
+                </div>
+                <div *ngFor="let cart of quotes?.carts" class="col-4">
+                  <span class="link px-0" (click)="retrieveQuote(cart)">
+                    {{ cart.code }}
+                  </span>
+                </div>
+              </li>
+            </ul>
+          </cx-fs-accordion-item>
+          <cx-fs-accordion-item
+            *ngIf="
+              billingEventLabels.length > 0 && categoryConfig.billingEvents
+            "
+            [opened]="false"
+            title="{{ 'quote.quoteBillingEvents' | cxTranslate }}"
+          >
+            <ul class="accordion-list container-fluid">
+              <li class="accordion-list-item row sub-title">
+                <div class="col-4">
+                  {{ 'quote.billingEvents' | cxTranslate }}
+                </div>
+                <div *ngFor="let cart of quotes?.carts" class="col-4">
+                  {{
+                    ((cart?.deliveryOrderGroups)[0]?.entries)[0]?.product
+                      ?.cartDisplayName
+                  }}
+                </div>
+              </li>
+              <ng-container
+                *ngFor="let billingEventLabel of billingEventLabels"
+              >
+                <li class="accordion-list-item row">
+                  <div class="item-label" class="col-4">
+                    {{ billingEventLabel }}
+                  </div>
+                  <ng-container *ngFor="let cart of quotes?.carts">
+                    <div class="col-4 value notice">
+                      {{
+                        (cart?.entries)[0]?.product?.price
+                          ?.oneTimeChargeEntries
+                          | cxFsBillingEventValue: billingEventLabel
```

Comment:
I created pure pipe for returning billable event value instead of calling method in template since it is better for performance:

[https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496](url)